### ISSUE TITLE
Support new versions of ruby

### ISSUE THREAD

**Issue by adisos** [SublimeLinter/SublimeLinter-ruby#12]
>Ruby 2.0 introduced first-class support for keyword arguments:
>
>```
>def foo(bar: 'default')
>  puts bar
>end
>
>foo # => 'default'
>foo(bar: 'baz') # => 'baz'
>```
>Ruby 2.0 blocks can also be defined with keyword arguments:
>```
>define_method(:foo) do |bar: 'default'|
>  puts bar
>end
>
>foo # => 'default'
>foo(bar: 'baz') # => 'baz'
>```
>
>Ruby 2.1 introduced required keyword arguments, which are defined with a trailing colon:
>```
>def foo(bar:)
>  puts bar
>end
>
>foo # => ArgumentError: missing keyword: bar
>foo(bar: 'baz') # => 'baz'
>```

**Comment by adisos** [comment]
SublimeLinter uses `ruby -wc` under the hood, so my `PATH` was incorrect. It used old 1.9.3 version, instead of RVM. 
I changed sublime-linter's config:
```
"paths": {
    "linux": [
        "~/.zshenv"
    ],
    "osx": [],
    "windows": []
}
```


~/.zshenv:
```
[[ -s "$HOME/.rvm/scripts/rvm" ]] && source "$HOME/.rvm/scripts/rvm"
```

**Comment by Nakilon** [comment]
macOS, rbenv
```json
{
  "lint_mode": "load_save",
  "show_panel_on_save": "view",
  "paths": {
    "linux": [],
    "osx": ["~/.zshrc"],
    "windows": []
  },
}
```
does not work, still uses default ruby.  
In project folder the version is configured via `rbenv local`.

**Comment by Nakilon** [comment]
nvm, sorry for false alarm
```
    "osx": ["~/.rbenv/shims"],
```
this works
